src/librustdoc/html/render/mod.rs

```diff
@@ -901,8 +903,13 @@ fn render_assoc_item(
             }
         };
         let vis = meth.visibility.print_with_space(meth.def_id, cx).to_string();
-        let constness =
-            print_constness_with_space(&header.constness, meth.const_stability(cx.tcx()));
+        // FIXME: Once https://github.com/rust-lang/rust/issues/67792 is merged, we can remove
+        // this condition.
+        let constness = if render_mode == RenderMode::Normal {
```

Comment:
Please use an exhaustive pattern match here. It's unclear just from looking at the code why this conditional is here, and a pattern match would make it clearer.

Reply:
Good point.